tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+
+class BottomBar(Base):
```

Comment:
And this BottomBar class could use an __init__ method to make sure that the BottomBar is ready and displayed, I think (to prevent intermittent failures).